client/ui/qml/Controls2/TextFieldWithHeaderType.qml

```diff
@@ -7,6 +7,8 @@ import "TextTypes"
 Item {
     id: root
 
+    signal sig_next()
```

Comment:
Wouldn't it be better to use https://doc.qt.io/qt-6/qml-qtquick-keynavigation.html to navigate with hotkeys?